src/main/drivers/osd_symbols.h

```diff
@@ -234,6 +234,8 @@
 #define SYM_AH_CH_CENTER            0x166 // 358 Crossair center
 #define SYM_FLIGHT_DIST_REMAINING   0x167 // 359 Flight distance reminaing
 #define SYM_ODOMETER                0x168 // 360 Odometer
+#define SYM_RX_BAND                 0x169 // 361 RX Band
```

Comment:
New symbols? Just in time. ;)

Reply:
Shannon is also informed and will add them to his HD font sets.

Reply:
Got to keep those DJI folks on their toes. Shannon has added the symbols to his cool fonts.